src/dialog/windows/SDL_windowsdialog.c

```diff
@@ -34,29 +34,49 @@
 typedef struct
 {
     bool is_save;
-    const SDL_DialogFileFilter *filters;
-    int nfilters;
-    const char* default_file;
+    wchar_t *filters_str;
+    char* default_file;
```

Comment:
```suggestion
    char *default_file;
```
SDL style is to have `*` next to the variable name. Can you adjust the code below as well?

Reply:
I've realized that I did that mistake not only in the commit but also in the previously written code. Thanks regexes for making my correcting job easy :^)

I've kept it in a separate commit to avoid putting too many changes in a single commit, but I can squash them if necessary.